README

```diff
@@ -78,10 +78,42 @@ and open the online version instead when requested.
 Installing from a Git clone
 +++++++++++++++++++++++++++
 
-Install Autotools (*automake*, *autoconf* and *libtool*), *intltool*,
-and the GLib development files **before** running any of the following
-commands, as well as *rst2html* from Docutils (see above for details).
-Then, run ``./autogen.sh`` and then follow the instructions for
+Using the Meson build system
+++++++++++++++++++++++++++++
+
+N.B. Meson support is still incomplete and a work-in-progress.
```

Comment:
I guess this could be removed before merge?

Reply:
Perhaps it should say "N.B. Meson support is an initial version for testing" at merge.